chart/templates/core/storageclass.yaml

```diff
@@ -10,6 +10,17 @@ provisioner: csi.hetzner.cloud
 volumeBindingMode: WaitForFirstConsumer
 allowVolumeExpansion: true
 reclaimPolicy: {{ $val.reclaimPolicy | quote }}
+{{- if $val.includesRobotServers }}
+allowedTopologies:
+- matchLabelExpressions:
+  - key: instance.hetzner.cloud/is-root-server
+    values:
+    - "false"
```

Comment:
With the new labels, I would just remove `is-root-server=false`

Reply:
Should we also remove the deprecation notice from the docs?
https://github.com/hetznercloud/csi-driver/blob/052842a3f39341e7db4413da30c7a5383369593f/docs/kubernetes/README.md?plain=1#L219-L231

Reply:
Makes sense to at least state that we now prefer the new label, even though there is no immediate reason to remove the old label.